respondd.py

```diff
@@ -69,6 +69,7 @@ def handle(self):
         ("", args.port),
         get_handler(get_providers(args.directory), {'batadv_dev': args.batadv_iface, 'mesh_ipv4': args.mesh_ipv4})
     )
+    server.daemon_threads = True
```

Comment:
Is this enough? How do the threads behave on exit of the main application? Are the terminated as well or do we need to add additional logic to ensure thread termination?

Reply:
Hm? It just restores the old default behavior of ThreadingMixIn. And daemon_threads set to true will make the threads behave autonomously. So you don't need to wait for these threads.

Reply:
Ah, ok. Should have read your issue description more carefully. Patch looks good to me.